_posts/2022-02-16-cockpit-263.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Cockpit 263
+author: garrett
+date: '2022-02-16'
+tags: cockpit
+slug: cockpit-263
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 263:
+
+
+
+## The back and forward buttons in your browser now work as expected
+
+Cockpit used to interfere with the browser history, and hitting the back button would frequently get into a loop, for example. This has been fixed and the back and forward button are now enabled again in Cockpit Client.
+
+
+## Try it out
+
+Cockpit 263 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/263)
+* [Cockpit Fedora 36](https://bodhi.fedoraproject.org/updates/?releases=F36&packages=cockpit)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
```

Comment:
Just to make sure, please update this to the actual bodhi URLs.

Reply:
Done.